python/paddle/distributed/fleet/base/distributed_strategy.py

```diff
@@ -189,38 +368,50 @@ def __init__(self):
         DistributedStrategy supports configurations from BuildStrategy.
 
         """
-        self.strategy = distributed_strategy_pb2.DistributedStrategy()
+        self.strategy: DistributedStrategy = (
```

Comment:
```suggestion
        self.strategy: Any = (
```

这里的 `strategy` 与 `DistributedStrategy` 不是同一个东西，`distributed_strategy_pb2.DistributedStrategy` 是 `protobuf` 生成的，这里直接用 `Any` 表示吧 ～